lib/experiments-page.php

```diff
@@ -84,6 +84,17 @@ function gutenberg_initialize_experiments_settings() {
 			'id'    => 'gutenberg-full-site-editing',
 		)
 	);
+	add_settings_field(
+		'gutenberg-full-site-editing-demo',
```

Comment:
I don't think we need an extra flag for this. I'd suggest the following:
If the base flag for full size editing is enabled and the current theme as well as its child theme don't offer `/Block-Templates/index` (the minimum requirement), rely on the demo templates automatically. Full size editing without a theme supporting it and without a demo would provide a bad user experience.